### PR TITLE
Added Code of Conduct plus doc updates

### PR DESCRIPTION
**Pull request by samgnaniah**
Added Code of Conduct for Ballerina project and made some more changes to the Quick Tour and BBE content.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
 [![GitHub last commit](https://img.shields.io/github/last-commit/ballerina-lang/ballerina.svg)](https://github.com/ballerina-lang/ballerina/commits/master)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-Ballerina is a simple programming language whose syntax and runtime addresses the integration problems in an agile way. Ballerina is designed to make it easier to write programs that integrate with data sources, services, and network-connected APIs of all kinds. It is optimized primarily for such programs. While it can be used to program anything, it is not recommended to use Ballerina if a significant portion of the program is not related to integrating with data sources, services, or network-connected APIs. 
+Ballerina makes it easy to create resilient services that integrate and orchestrate transactions across distributed endpoints. 
+
+Ballerina is created for and dedicated to developers who want to solve integration problems. Ballerina is inspired by Java, Go, and other languages, and leverages years of experience in the integration domain.
```
Comment (TylerJewell) [inline README.md]: I don't think this paragraph is helping - would remove it.

Comment (samgnaniah) [reply README.md]: Changed it to have only this content as the intro:

"Ballerina allows you to code with a statically-typed, interaction-centric programming language where microservices, APIs, and streams are first-class constructs. You can use your preferred IDE and CI/CD tools. Discover, consume, and share packages that integrate endpoints with Ballerina Central. Build binaries, containers, and Kubernetes artifacts and deploy as chaos-ready services on cloud and serverless infrastructure. Integrate distributed endpoints with simple syntax for resiliency, circuit breakers, transactions, and events."

Does this sound ok?

**Comment by TylerJewell** [comment]
@hasithaa - why did we merge this PR?  I was a reviewer and had indicated that I was not ready to give +1 - there were improvements needed.  Do we reopen this PR or have to do a new one?

**Comment by hasithaa** [comment]
@TylerJewell - @uthaiyashankar asked to merge this. I guess he has verified with Sam. Let me revert this PR. @samgnaniah Can you open a new PR ?

**Comment by hasithaa** [comment]
@samgnaniah, I can't revert this PR. Github gives following error message. 

"this pull request couldn’t be reverted automatically. It may have already been reverted, or the content may have changed since it was merged."

Please send a new PR with updated content.

**Pull request merged**